applications/StructuralMechanicsApplication/python_scripts/eigen_solution_input_process.py

```diff
@@ -0,0 +1,42 @@
+import KratosMultiphysics
+import KratosMultiphysics.HDF5Application as KratosHDF5
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+
+def Factory(settings, Model):
+    if not isinstance(settings, KratosMultiphysics.Parameters):
```

Comment:
I would check with ´type´ explicitly since that is whats done everywhere else in Kratos